frontend/talentsearch/src/js/components/reviewMyApplication/ReviewMyApplicationPage.tsx

```diff
@@ -0,0 +1,266 @@
+import { commonMessages } from "@common/messages";
+import { getLocale } from "@common/helpers/localize";
+import MissingSkills from "@common/components/skills/MissingSkills";
+import { notEmpty } from "@common/helpers/util";
+import ExperienceSection from "@common/components/UserProfile/ExperienceSection";
+import { Link } from "@common/components";
+import { getMissingSkills } from "@common/components/skills/MissingSkills/MissingSkills";
+import {
+  Applicant,
+  Maybe,
+  PoolAdvertisement,
+  useGetReviewMyApplicationPageDataQuery,
+} from "../../api/generated";
+import { useDirectIntakeRoutes } from "../../directIntakeRoutes";
+import ApplicationPageWrapper from "../ApplicationPageWrapper/ApplicationPageWrapper";
+import { flattenExperienceSkills } from "../experienceAndSkills/ExperienceAndSkills";
+
+interface ReviewMyApplicationProps {
+  applicant: Applicant;
+  poolAdvertisement: PoolAdvertisement;
+  poolCandidateId: string;
+  closingDate: Date;
+  jobTitle: Maybe<string>;
+  poolId: string;
+}
+
+export const ReviewMyApplication: React.FunctionComponent<
+  ReviewMyApplicationProps
+> = ({
+  applicant,
+  poolAdvertisement,
+  poolCandidateId,
+  closingDate,
+  jobTitle,
+  poolId,
+}) => {
+  const intl = useIntl();
+  const directIntakePaths = useDirectIntakeRoutes();
+  const experiences = applicant.experiences?.filter(notEmpty) || [];
+  const missingSkills = {
+    requiredSkills: poolAdvertisement.essentialSkills?.filter(notEmpty),
+    optionalSkills: poolAdvertisement.nonessentialSkills?.filter(notEmpty),
+  };
+  const hasExperiences = notEmpty(applicant.experiences);
+  const { isProfileComplete } = applicant;
+  const isApplicationComplete =
+    isProfileComplete ||
+    getMissingSkills(
+      missingSkills.requiredSkills || [],
+      hasExperiences ? flattenExperienceSkills(experiences) : [],
+    ).length === 0;
+
+  return (
+    <ApplicationPageWrapper
+      closingDate={closingDate}
+      title={intl.formatMessage({
+        defaultMessage: "My application profile",
+        description: "Title for review application page.",
+      })}
+      crumbs={[
+        {
+          title: intl.formatMessage({
+            defaultMessage: "My applications",
+            description: "Breadcrumb for review application page.",
+          }),
+          href: directIntakePaths.allPools(),
+        },
+        {
+          title:
+            jobTitle || intl.formatMessage(commonMessages.jobTitleNotFound),
+          href: directIntakePaths.poolApply(poolId),
+        },
+        {
+          title: intl.formatMessage({
+            defaultMessage: "Step 1",
+            description: "Breadcrumb for review application page.",
+          }),
+        },
+      ]}
+      navigation={{
+        currentStep: 1,
+        steps: [
+          {
+            path: directIntakePaths.reviewApplication(applicant.id),
+            label: intl.formatMessage({
+              defaultMessage: "Step 1: Review my profile",
```

Comment:
The "Review my profile" part of this string should be heavy. 
![image](https://user-images.githubusercontent.com/8978655/187976877-377496aa-e7c3-46dc-bfcd-8d5af3787769.png)

Reply:
Fixed by [Improve styling to match designs](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3796/commits/4c06217b89c30bf6548078c889c46d6de10ca560)